pkg/gatewayserver/gatewayserver.go

```diff
@@ -371,16 +402,16 @@ func (gs *GatewayServer) handleUpstream(conn *io.Connection) {
 						drop(ttnpb.EndDeviceIdentifiers{}, err)
 						break
 					}
-					ns := gs.GetPeer(ctx, ttnpb.PeerInfo_NETWORK_SERVER, ids)
-					if ns == nil {
-						drop(ids, errNoNetworkServer)
+					handler := item.host.handler(&ids)
+					if handler == nil {
+						drop(ids, errNoForwardToHost.WithAttributes("host", item.host.name))
```

Comment:
Doesn't this mean that we'll register drops for every packet broker that isn't configured to handle a specific DevAddr? Wouldn't those drop events confuse users?

Reply:
Yeah that's indeed the implication. Indeed, that's generating _a lot_ of events that are a result of specific configuration; by default, everything gets forwarded to the cluster NS.

I'll remove this call.